dex/src/test/java/io/crate/testing/RowGenerator.java

```diff
@@ -38,37 +38,31 @@
      * @param to   end (exclusive)
      */
     public static Iterable<Row> range(final int from, final int to) {
-        return new Iterable<Row>() {
-
-            @Override
-            public Iterator<Row> iterator() {
-                return new Iterator<Row>() {
+        return () -> new Iterator<Row>() {
```

Comment:
Is this actually better? Maybe it's a personal thing but new Iterable<Row> is more readable than () ->